app/models/mazeInfo/MazeDataHolder.scala

```diff
@@ -0,0 +1,15 @@
+package models.mazeInfo
+
+case class MazeDataHolder(
+    mazeDimensions: MazeDimensions = MazeDimensions(70, 40),
+    bgHexColor: String = "#4b4b4b",
+    fgHexColor: String = "#c8c8c8",
+    solved: Boolean = false){
+
+    val bgIntColor: Int = Integer.parseInt(bgHexColor.substring(1), 16)
```

Comment:
Why do we define as string then convert to Int?

It feels weird to put the type in the name of the variable. I know it's an int cos it's defined as one

Reply:
That's the default value if the argument is not supplied to the case class. The idea is that, in the case where hex color is supplied, it is automatically converted. If not there is one source of truth for the default color which is then converted to int format. 

Intellij says type definitions are required for public members so just went with it.